packages/builder/src/components/start/AppRow.svelte

```diff
@@ -18,14 +18,26 @@
   export let deleteApp
   export let unpublishApp
   export let releaseLock
+  export let editIcon
+
+  $: color =
+    $apps.find(filtered_app => app?.appId === filtered_app.appId)?.icon
+      ?.color || ""
+  $: name =
+    $apps.find(filtered_app => app?.appId === filtered_app.appId)?.icon?.name ||
```

Comment:
Same as above about just accessing the value directly - I think we can just use `app.name` directly.